looper/conductor.py

```diff
@@ -767,6 +772,31 @@ def _reset_curr_skips(self):
         self._curr_skip_pool = []
         self._curr_skip_size = 0
 
+    def check_executable_path(self, pl_iface):
+        """Determines if supplied pipelines are callable.
+        Raises error and exits Looper if not callable"""
+        pl_iface = pl_iface
+        pipeline_commands = []
+        if "path" in pl_iface.keys():
+            pipeline_commands.append(pl_iface["path"])
+        if (
+            "var_templates" in pl_iface.keys()
+            and "pipeline" in pl_iface["var_templates"].keys()
```

Comment:
this line will fail if `"var_templates"` are not in pl_iface

Reply:
In this case you have to add two if statements

Reply:
Or you can just do something like that:
```
try:
   if "pipeline" in pl_iface["var_templates"].keys() ....
except Exception: 
  pass
```

Reply:
We discussed that this two part check is working as is and not failing if var_templates does not exist, i.e. no key error.